typespec-tests/src/main/java/com/_specs_/azure/core/basic/BasicAsyncClient.java

```diff
@@ -541,10 +541,10 @@ public PagedFlux<User> list(Integer top, Integer skip, List<String> orderBy, Str
             }
         }
         PagedFlux<BinaryData> pagedFluxResponse = list(requestOptions);
-        return PagedFlux.create(() -> (continuationTokenParam, pageSizeParam) -> {
-            Flux<PagedResponse<BinaryData>> flux = (continuationTokenParam == null)
```

Comment:
I will check where the name change come from, but I think it will not have functional impact.

Reply:
This should be from a recent PR of Alan.